Runtime/Scripts/Snapshots.cs

```diff
@@ -149,24 +118,17 @@ private string DetermineSaveLocation()
             {
                 outputFileName = $"Snapshot_{DateTime.Now:yyyy-MM-dd_HH-mm-ss}";
             }
-            outputFileName = $"{outputFileName}.{FileType}";
+            string location = Application.persistentDataPath;
 
-            string location = Application.persistentDataPath
-                + Path.DirectorySeparatorChar
+#if UNITY_WEBGL && !UNITY_EDITOR
+            outputFileName = $"{outputFileName}.{FileType}";
+            location += Path.DirectorySeparatorChar
                 + targetPath
                 + Path.DirectorySeparatorChar
                 + outputFileName;
-
-#if UNITY_EDITOR
-            // Window for user to input desired path/name/filetype
-            location = EditorUtility.SaveFilePanel(
-                "Save texture as file",
-                "",
-                outputFileName,
-                fileType.ToString()
-            );
+#else       
+            StandaloneFileBrowser.SaveFilePanel("Save texture as file", location, outputFileName, FileType.ToString());
```

Comment:
Kan je niet beter location weglaten of een lege string? Nu stuur je de download panel direct naar de AppData folder en het is beter als ze daar niet direct naar gestuurd worden